src/System.IO.UnmanagedMemoryStream/tests/UmsTests.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Xunit;
+using System.Threading;
 
 // TODO: add WriteAsync, Timeout, Flush, CopyTo tests
```

Comment:
WriteAsync (mostly) and Flush included in this PR.
Timeout and CopyTo are not overloaded by `UnmanagedMemoryStream`, so what the reason to put their test here?